torch/_inductor/codegen/wrapper.py

```diff
@@ -2167,11 +2167,11 @@ def fill_args(arg, arg_type):
                         new_int_args.extend([str(a) for a in arg])
                 else:
                     assert isinstance(
-                        arg_type.getElementType(), static_arg_types
+                        arg_type.getElementType(), static_arg_types  # type: ignore[arg-type]
```

Comment:
Surprised by this noqa. Is it inferring the type incorrectly for static_arg_types? Can wrap it in a tuple or something to make it happy?

Reply:
This is what I was referring to in the commit message:

> The insinstance checks in wrapper.py reference torch.Type, which seems to cause mypy to choke. Not entirely sure why; I've just added type-ignore comments for now.

Deleting `torch.Type` from the `static_arg_types` tuple makes the error go away. Otherwise mypy complains that we are using a "typing special form"...